### ISSUE TITLE
Figure tags incorrectly wrapped with paragraph tags

### ISSUE THREAD

**Issue by bencrowder** [michelf/php-markdown#111]
Currently, using a <figure> tag will result in the tag being incorrectly wrapped in &lt;p&gt; tags. I added "figure" to the list on line 1729 ($block_tags_re) and it now works. (I also noticed that on line 247, maybe "figcaption" should be added to that list as well?)

**Comment by michelf** [comment]
You're right. This works fine with the regular Markdown parser but not with MarkdownExtra.
